docs/source/peer_ledger_snapshot.md

```diff
@@ -0,0 +1,164 @@
+* While endorsements will continue and queries can be submitted, all peers taking a snapshot at a particular height will not commit blocks on the channel while the snapshot is being generated. Because taking a snapshot is a resource-intensive operation, the peer might also be slow to endorse transactions or commit blocks on other channels. For these reasons, it is anticipated that snapshots will only be taken when necessary (for example, when a new peer is joining a channel or when organizations want to verify that no ledger forks have occurred). As long as every organization taking a snapshot has at least one peer that is not taking a snapshot, the operation of your organization and your channels should not be affected.
+* Because the private data between organizations in a channel is likely to be at least somewhat different, private data is not included in the snapshot (hashes of the private data are included, but not the data itself). Peers that join a channel using a snapshot will discover the collections it is a member of and pull the relevant private data from peers that are members of those collections directly.
+* The snapshot process is not meant to be a method to archive and prune the ledgers of peers that are already joined to the channel. Similarly, it is not meant as a method to take a full backup a peer, as peer configuration information, such as MSPs, are not included in a snapshot).
+
+Because of these limitations, the decision of whether a peer will join a channel from a snapshot depends on a few factors.
+
+* **The current ledger height of the channel**. The smaller the ledger, the less useful it is to join a channel using a snapshot. Where the size of the ledger crosses a point where it makes sense to use a snapshot depends on your use case.
+* **The importance of querying old data**. Some channels and use cases might call for querying old blocks regularly, while in other channels it might only be necessary to know the current state. This must be balanced by the server costs experienced by your organization.
+
+## Overview
+
+Snapshots can be used by organizations that already have peers on a channel or by organizations new to a channel. Whatever the use case, the process is largely the same.
+
+If the organization is new to the channel, the organizations currently joined to the channel must:
+
+1. **Schedule a snapshot**. These snapshots must be taken at **exactly the same ledger height** on each peer. This will allow the new peer organization to evaluate the snapshots to make sure they contain the same data. This ledger height must be equal or higher than the current ledger height. They cannot be taken from a lower ledger height. If you attempt to schedule a snapshot at a height lower than the current height you will get an error. Note that a peer that already used a snapshot to join a channel can also be used to take a snapshot.
```

Comment:
The text here implies that snapshots are taken as needed. But some networks may simply have a policy to snapshot peers at a regular cadence, for example every 10,000 blocks. That way consistent and recent snapshots are always available. It is worth mentioning as a consideration, either here or elsewhere. 

Note - it is not possible to schedule recurring snapshots (that may be a future enhancement). Each snapshot has to be scheduled independently. Although you can schedule as many future snapshots as you like.